vm/src/stdlib/grp.rs

```diff
@@ -0,0 +1,113 @@
+pub(crate) use grp::make_module;
+
+#[pymodule]
+mod grp {
+    use std::ptr::NonNull;
+
+    use crate::{
+        builtins::{PyIntRef, PyListRef, PyStrRef},
+        convert::{IntoPyException, ToPyObject},
+        exceptions,
+        types::PyStructSequence,
+        AsObject, PyObjectRef, PyResult, VirtualMachine,
+    };
+    use nix::unistd;
+
+    #[pyattr]
+    #[pyclass(module = "grp", name = "struct_group")]
+    #[derive(PyStructSequence)]
+    struct Group {
+        gr_name: String,
+        gr_passwd: String,
+        gr_gid: u32,
+        gr_mem: PyListRef,
+    }
+    #[pyimpl(with(PyStructSequence))]
+    impl Group {
+        fn from_unistd_group(group: unistd::Group, vm: &VirtualMachine) -> Self {
+            let cstr_lossy = |s: std::ffi::CString| {
+                s.into_string()
+                    .unwrap_or_else(|e| e.into_cstring().to_string_lossy().into_owned())
+            };
+            Group {
+                gr_name: group.name,
+                gr_passwd: cstr_lossy(group.passwd),
+                gr_gid: group.gid.as_raw(),
+                gr_mem: vm
+                    .ctx
+                    .new_list(group.mem.iter().map(|s| s.to_pyobject(vm)).collect()),
+            }
+        }
+    }
+
+    // impl From<unistd::Group> for Group {
+    //     fn from(group: unistd::Group) -> Self {
+    //         // this is just a pain...
+    //         let cstr_lossy = |s: std::ffi::CString| {
+    //             s.into_string()
+    //                 .unwrap_or_else(|e| e.into_cstring().to_string_lossy().into_owned())
+    //         };
+    //         Group {
+    //             gr_name: group.name,
+    //             gr_passwd: cstr_lossy(group.passwd),
+    //             gr_gid: group.gid.as_raw(),
+    //             gr_mem: group.mem,
+    //         }
+    //     }
+    // }
+
```

Comment:
```suggestion
```

Reply:
I forgot I didn't delete this, thank you